videoframes.py

```diff
@@ -206,6 +221,22 @@ def change_stream_quality(self, quality):
         self.stream.change_stream_quality(quality)
         self.player.play()
 
+    def open_stream_in_browser(self, event):
+        url = self.stream.url.lower()
+        url = ("http://www." if "www" not in url else "http://") + url
```

Comment:
E.g. right now all urls will be prepended with "http://", which can generate a broken url with the scheme duplicated. The urllib library will help you with the parsing 👍